### PR TITLE
Add a build var that disables SQ tasks in PR builds

### PR DESCRIPTION
**Pull request by bgavrilMS**
Tracking ticket https://jira.sonarsource.com/browse/TBSTFNP-47

### CODE REVIEW

File: Tasks/SonarQubePostTest/SonarQubePostTest.ps1 [hunk 1]
```diff
@@ -2,11 +2,19 @@ Write-Verbose "Starting SonarQube PostBuild Step"
...
+	return
+}
+
+. $PSScriptRoot/SonarQubePostTestImpl.ps1
+. $PSScriptRoot/SonarQubeBuildBreaker.ps1
+. $PSScriptRoot/CodeAnalysisFilePathComputation.ps1
 
 InvokeMSBuildRunnerPostTest
 UploadSummaryMdReport
+HandleCodeAnalysisReporting
```
Comment (bgavrilMS) [inline Tasks/SonarQubePostTest/SonarQubePostTest.ps1]: This statement is part of the original work to enable PR-CA and should appear on the left as well.

File: Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1 [hunk 2]
```diff
@@ -25,8 +25,15 @@ Write-Verbose "breakBuild = $breakBuild"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 
-. ./Common/SonarQubeHelpers/SonarQubeHelper.ps1
-. ./SonarQubePreBuildImpl.ps1
+. $PSScriptRoot/Common/SonarQubeHelpers/SonarQubeHelper.ps1
```
Comment (bgavrilMS) [inline Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1]: I'm converting all script imports to use $PSScriptRoot as the tests will be able to import the product scripts as well.

**Comment by HSAR** [comment]
LGTM

**Comment by duncanpMS** [comment]
LGTM

**Pull request merged**